source/solvers/gls_sharp_navier_stokes.cc

```diff
@@ -66,6 +68,39 @@ GLSSharpNavierStokesSolver<dim>::vertices_cell_mapping()
     }
 }
 
+template <int dim>
+void
+GLSSharpNavierStokesSolver<dim>::cutted_cells_mapping()
+{
+  // check all the cells if they are cut or not. Put the information in a map
+  // with the key being the cell.
+  TimerOutput::Scope t(this->computing_timer, "cutted_cells_mapping");
+  std::map<types::global_dof_index, Point<dim>> support_points;
+  DoFTools::map_dofs_to_support_points(*this->mapping,
+                                       this->dof_handler,
+                                       support_points);
+  cutted_cells_map.clear();
+  const auto &       cell_iterator = this->dof_handler.active_cell_iterators();
+  const unsigned int dofs_per_cell = this->fe->dofs_per_cell;
+  std::vector<types::global_dof_index> local_dof_indices(dofs_per_cell);
+
+  // // Loop on all the cells and check if they are cut.
+  for (const auto &cell : cell_iterator)
+    {
+      if (cell->is_locally_owned() || cell->is_ghost())
+        {
+          bool         cell_is_cut;
+          unsigned int p;
```

Comment:
What is this p? I don't get it. You make a tuple with it but what is it for?
Why is it often set to std::ignore? I don't get it.... Either you can remove it or you need to explain what it is for...

Reply:
I added the information in the code. p is the particle id of the particle that cut the cell. this info is only used in the sharp edge function. All other calls to this map only need to know if the cell is cut or not and doesn’t care about what particle cuts it. In these cases I used std::ignore.

Reply:
Then call it particle_id or ib_particle_id or sharp_particle_id, but not p! :)! P does not mean anything to a normal reader